jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    "roots": [
+        "<rootDir>/src"
+    ],
+    "transform": {
+        "^.+\\.ts$": "ts-jest"
```

Comment:
You can use babel-jest, in this case your babel config will involved

Reply:
`babel-jest` instead of `ts-jest`?

Reply:
yep

Reply:
done